CI-test/Foo.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Foo.swift
+//  CI-test
+//
+//  Created by Eugen Spinu on 10/01/2021.
+//
+
+import Foundation
+
+struct Foo {
+    let param1: Int
+    let param2: Int
+
+    func foo() -> Int {
+        _ = foo1()
+        let bla = "unused variable"
+        return param1+param2
```

Comment:
<table data-meta="generated_by_metrics">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/espinu/iOS-CI-test/blob/ce3718d6a751ad70543283daa1513d92742586b4/CI-test/Foo.swift#L17"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>Foo.swift:17</code></td>
    </tr>
  </tbody>
</table>